provision/golang.sh

```diff
@@ -6,7 +6,8 @@ set -e
 
 sudo -u vagrant -E bash -c "mkdir ${GOPATH} && \
 go get -u github.com/google/gops && \
-go get github.com/subfuzion/envtpl/... && \
-go get -u github.com/gordonklaus/ineffassign"
+go get github.com/subfuzion/envtpl/... "
+
+curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sudo sh -s -- -b ${GOPATH}/bin/ v1.27.0
```

Comment:
Why not use `got get` here? `go get github.com/golangci/golangci-lint/cmd/golangci-lint@v1.27.0` according to the doc.

Reply:
Oh nice, I didn't know that 1.27 is supporting go get now. Last time, I was not recommended https://github.com/golangci/golangci-lint/issues/873

Let me make a change on this :). Thanks for your input :100:

Reply:
@sayboras The latest version of this PR still seems to be using `curl`. Are you intending to change it to use `go get` as suggested?

Reply:
@tklauser See https://github.com/cilium/packer-ci-build/pull/211#issuecomment-633197689. Using curl is actually the recommended way.